pom.xml

```diff
@@ -95,9 +95,10 @@
         <version.org.apache.ds>2.0.0-M15</version.org.apache.ds> <!-- TODO Elytron - Bump to M17 -->
         <version.org.apache.httpcomponents.httpclient>4.5.2</version.org.apache.httpcomponents.httpclient>
         <version.org.apache.httpcomponents.httpcore>4.4.4</version.org.apache.httpcomponents.httpcore>
-        <version.org.apache.maven>3.3.9</version.org.apache.maven>
+        <version.org.apache.maven>3.5.0</version.org.apache.maven>
```

Comment:
Have we tested with maven 3.3.1 as that is the current minimum requirement? I just want to make sure we don't have any weird clashes.

Reply:
@ctomc Can you please provide more info why this change is needed ?

Reply:
@jamezp @psakar this version of maven here has nothing to do with requirements for building with maven 3.5 it is only for dependency management of [maven-resolver](https://maven.apache.org/resolver/)  (ex eclipse aether) which we use in subsystem legacy test framework to be able to load legacy version of extensions when testing transformers.

Reply:
Right yeah. I was more worried about ending up with a mismatch on the class path for tests is all. Those tests use some custom class loading, plus surefire itself has it's own class loading issues at times. I have a feeling it will work, but I just think we should test it.

Reply:
It does work, it is tested, if testsuite passes that is a huge test.

CI uses maven 3.3.9, locally I am using maven 3.5 and it works.

And yeah, this stuff ends up in custom classloader completely.